### DISCUSSION TITLE
Filter Settings

### DISCUSSION THREAD

**Discussion by oguzhanyediel** [apache/superset#16705] [General]
Hello everyone,
I have a question about the filtering system. Why new-added filter effects all the other graphs in any dashboard?

For example, I add a new line graph to any dashboard, and then, I want to add a filter box by using the same data source of the line graph. When I add the filter box, it affects the other bar graph or line graph. I don't understand why that is. Is it a default feature?

My Superset Version: **0.999.0dev**

**Comment by kamalkeshavani-aiinside** [comment]
Yes, by default the filter scope is of the same tab. If you are not using tabs, then the scope is root, which refers to the whole dashboard.
You will have to change the filter mapping for every filter you add to a dashboard as per your needs.
